Link libmachuser and libhurduser automatically with libc, since they are
considered a standard part of the API in GNU-land.
See <http://lists.gnu.org/archive/html/bug-hurd/2011-03/msg00112.html>

---
 Makerules |    3 +++
 1 file changed, 3 insertions(+)
--- a/Makerules
+++ b/Makerules
@@ -1001,6 +1001,9 @@ $(inst_libdir)/libc.so: $(common-objpfx)
 	      '$(libdir)/$(patsubst %,$(libtype.oS),$(libprefix)$(libc-name))'\
 	      ' AS_NEEDED (' $(rtlddir)/$(rtld-installed-name) ') )' \
 	) > $@.new
+ifeq ($(patsubst gnu%,,$(config-os)),)
+	echo 'INPUT ( AS_NEEDED ( -lmachuser -lhurduser ) )' >> $@.new
+endif
 	mv -f $@.new $@
 
 endif
